ripgrep-feedstock/recipe/meta.yaml

```diff
@@ -19,9 +19,20 @@ build:
     - cp target/release/rg $PREFIX/bin/rg  # [not win]
     - copy target\release\rg.exe %PREFIX%\Library\mingw-w64\bin\rg.exe  # [win]
   missing_dso_whitelist:
-    - /usr/lib/libresolv.9.dylib
+    - $RPATH/ld64.so.1  # [s390x]
 
 requirements:
   build:
     - {{ compiler('rust') }}  # [not win]
     - {{ compiler('rust-gnu') }}  # [win]
+
+test:
+  commands:
+    - rg --version
+
+about:
+  home: https://github.com/BurntSushi/ripgrep
+  summary: ripgrep is a line-oriented search tool that recursively searches the current directory for a regex pattern.
+  license: MIT
```

Comment:
Could you add also license_family: Other, dev_url, and doc_url?
And this project is dual-licensed under the `Unlicense` and `MIT` licenses, see https://github.com/BurntSushi/ripgrep/blob/master/COPYING

Reply:
We need to check if it will impact downstream packages conda-build, python, etc.